.travis.yml

```diff
@@ -22,8 +22,8 @@ sudo: false
 #dist:
```

Comment:
As comments in the template say, for C++11 you need trusty - try uncommenting these two lines. Usually docker should have been equivalent, though :\ Maybe with the recent changes in Travis images (done quarterly or so) something got changed too much.

In any case, in Travis logs where it installs APT packages, gcc should be not 4.6.x but at least 4.8.x.